spacy/tests/test_cli_app.py

```diff
@@ -101,12 +104,23 @@ def test_project_run(project_dir):
 
 
 def test_project_clone():
-    with make_tempdir() as workspace:
+    # Git clones create some readonly/hidden files. On Windows, trying to
+    # delete these gives an error, so we have to handle that specially.
```

Comment:
Would it make sense to add this to `make_tempdir` instead?

Reply:
Sure, that makes sense. In the narrow case of git clones it's clear why this is happening. In the more general case it could maybe mask a real issue, though I guess for `make_tempdir` it wouldn't matter.

Reply:
It turns out the changes here didn't fix the issue because it's in the `clone` command, not the test, anyway, so moving it to the general implementation fixes that.